BAZProjectC1/Control/ParallaxView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  ParallaxView.swift
+//  BAZProjectC1g
+//
+//  Created by efloresco on 02/10/22.
+//
+
+import Foundation
+import UIKit
+
+protocol ParalaxProtocol {
+    func didSelectItem(iIndex: Int)
+}
+
+class ParallaxView: UIView {
+    
+    var delegateParalax: ParalaxProtocol?
```

Comment:
Revisa si es necesario agregar weak a este delegado:

```suggestion
    weak var delegateParalax: ParalaxProtocol?
```